src/rock/models/feeds/resolver.js

```diff
@@ -4,19 +4,18 @@ export default {
   Query: {
     userFeed: async (
       _,
-      { filters, limit, skip, status, cache, options = "{}" },
+      { filters, limit, skip, status, cache },
       { models, person, user },
     ) => {
       if (!filters) return null;
 
-      const opts = JSON.parse(options);
       const filterQueries = [];
 
       // Home feed query
       if (filters.includes("CONTENT")) {
-        let { channels } = opts.content;
```

Comment:
@Lepozepo Would it be easy enough to just change this logic to be something like:

```
let channels = get(opts, 'content.channels');
if (!channels) {
  const topics = await models.User.getUserFollowingTopics(person && person.PrimaryAliasId);
  channels = topics
    .map ...
    ...
}

Reply:
For backwards compatibility? Yeah, that would work

Reply:
yes